templates/master/00-master/ovirt/files/ovirt-keepalived-keepalived.yaml

```diff
@@ -10,7 +10,7 @@ contents:
     }
 
     vrrp_script chk_dns {
-        script "/usr/bin/host -t SRV _etcd-server-ssl._tcp.{{ .EtcdDiscoveryDomain }} localhost"
+        script "/usr/bin/host api-int.{{ .EtcdDiscoveryDomain }} localhost"
```

Comment:
same as https://github.com/openshift/machine-config-operator/pull/1556/files#r433445452

Reply:
Same here, this check was removed in https://github.com/openshift/machine-config-operator/pull/1775